### PR TITLE
Create CHANGELOG.md

### PR DESCRIPTION
**Pull request by pjfanning**
relates to https://github.com/apache/incubator-pekko/issues/833

### CODE REVIEW

**Review by nvollmar — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Apache Pekko Releases
```
Comment (He-Pin) [inline CHANGELOG.md:1]: Release notes？

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+The [Release Notes](https://pekko.apache.org/docs/pekko/current/release-notes/index.html) are available on the Apache Pekko web site.
+
+## GitHub Compare
+
+GitHub has a feature allowing you to compare 2 Git tags and see the PRs and overall code diff. You can take the example URL below and tweak it with the version tags that you want to compare.
+
+https://github.com/apache/incubator-pekko/compare/v1.0.1...v1.0.2
```
Comment (He-Pin) [inline CHANGELOG.md:9]: I think a better way is we creating milestones for every release.

Comment (pjfanning) [reply CHANGELOG.md:9]: this is an exact copy of what I've put on other repos - we have release notes and I'm pointing to them 

I am a volunteer. I will not spend time creating multiple copies of release notes.

File: CHANGELOG.md (hunk 1 above)
Comment (pjfanning) [reply CHANGELOG.md:1]: these are not release notes - they are a link to our release notes - please see https://github.com/apache/incubator-pekko/issues/833 for the background

File: CHANGELOG.md (hunk 2 above)
Comment (He-Pin) [reply CHANGELOG.md:9]: Let's merge this first then, as any updates can come up after

**Review by He-Pin — APPROVED** [review]
Lgtm

**Pull request merged**